app/presenters/contact_gone_presenter.rb

```diff
@@ -15,6 +17,12 @@ def present
       routes: [
         { path: contact.link, type: "exact" }
       ],
+      content_id: content_id,
     }
   end
+
+  private
+  def content_id
+    @_conten_id ||= SecureRandom.uuid
```

Comment:
Typo here and in the equivalent method in `ContactRedirectPresenter`: `_conten_id`